src/Components/booksform.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const booksform = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <div className="App">
+      {categories}
+    </div>
+  );
+};
+
+export default booksform;
```

Comment:
- `booksform.js should be placed in folder `src/containers`
- should have a state with `title` and `category`
- it should renders form with title input, categories select box and submit [[screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/RenderForm.png)]